src/main/resources/com/google/api/codegen/py/python_gapic.yaml

```diff
@@ -9,3 +9,7 @@ templates:
   input_element_view: com.google.api.codegen.py.PythonProtoFileView
   snippet_files:
   - message.snip
+- codegen_provider: com.google.api.codegen.clientconfig.ClientConfigGapicProvider
+  input_element_view: com.google.api.codegen.InterfaceView
+  snippet_files:
+  - json.snip
```

Comment:
Something we might want to consider addressing (separate from this PR): now that we're mixing providers from different directories, it could be somewhat confusing to find the right snippet file in the source tree corresponding to the provider, because the rule connecting the two is implicit, and we have multiple snippet files named the same thing (so many main.snip's!). I think we should consider fully qualifying the paths to our snippet files in the config yaml's.